### PR TITLE
libX11: update to 1.8.4.

### PR DESCRIPTION
**Pull request by someone13574**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl (crossbuild)
  - armv6l (crossbuild)

### CODE REVIEW

File: srcpkgs/libX11/template [hunk 1]
```diff
@@ -14,7 +13,7 @@ maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="MIT"
 homepage="http://xorg.freedesktop.org/"
 distfiles="${XORG_SITE}/lib/${pkgname}-${version}.tar.xz"
```
Comment (mhmdanas) [inline srcpkgs/libX11/template:15]:
```suggestion
homepage="https://gitlab.freedesktop.org/xorg/lib/libx11"
distfiles="${XORG_SITE}/lib/$libX11-${version}.tar.xz"
```

**Comment by mhmdanas** [comment]
Remove the `do_configure` thing, it shouldn't be necessary I think.

**Comment by someone13574** [comment]
Can I get a CI rerun on this, x86_64-musl failed due to network timeout.

**Comment by dataCobra** [comment]
You can do the rerun by yourself.

Use `git push --force`.

**Pull request merged**